{
  "Header": {
    "Original": "CVE-2025-38612 (In the Linux kernel, the following vulnerability has been resolved:  s ...)",
    "ID": "CVE-2025-38612",
    "Description": "(In the Linux kernel, the following vulnerability has been resolved:  s ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-4328-1 DLA-4327-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-4328-1",
        "DLA-4327-1"
      ]
    },
    {
      "Original": "- linux 6.16.3-1",
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.16.3-1"
    },
    {
      "Original": "[trixie] - linux 6.12.43-1",
      "Type": "package",
      "Release": "trixie",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.12.43-1"
    },
    {
      "Original": "[bookworm] - linux 6.1.148-1",
      "Type": "package",
      "Release": "bookworm",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.1.148-1"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/eb2cb7dab60f9be0b435ac4a674255429a36d72c (6.17-rc1)",
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/eb2cb7dab60f9be0b435ac4a674255429a36d72c (6.17-rc1)"
    }
  ]
}